### ISSUE TITLE
Logging Client Activity on Server

### ISSUE THREAD

**Issue by Schabbach** [FreeOpcUa/python-opcua#533]
Hello,

is there a way to log the activity from a client?
For example, I'd like to know, when a client reads an object, reads a variable, writes a variable, connects to the server.
Further I'd like to know wich Variable or Object the client has read/changed.

Thanks for your help!

**Comment by oroulet** [comment]
No. Would need to add code for it. Maybe change a bit the architecture

**Comment by adityasriv22** [comment]
What does the server-events.py example do? there is no documentation on logging of client events.
